ana-febles.txt

```diff
@@ -0,0 +1,145 @@
+https://codepen.io/AnaFebles/pen/vYLrexy
+
+
+1. What is the difference between an inline element and a block element? 
+in line: it will not force a new line break, everything will be aling in the same line. we cannot put in block elements on in line ones. 
+block: will forcve a new line to be created.
+
+2. What happens when an element is positioned absolutely?
+it is a fixed psititon, basically it will not move from that place.
+
+3. How do I make an element take up only the amount of space it needs but also have the ability to give it a width? 
+div
+
+4. Name 3 elements that are diplay block by default, 2 elements that are display inline by default and 1 element that is display inline-block by default 
+
+block: div, header, section
+in line: span, img
+
+5. In your own words, explain the box model. What is the fix for the box model?
+everything in CSS is boxes, both in line and block. This helps understanding layout and how to align elements. 
```

Comment:
This isn't what the box model is. You can read up on the box model at the following links:
- <https://www.w3schools.com/Css/css_boxmodel.asp>
- <https://learn.lambdaschool.com/web1/module/rec6KMj0W3ukhYeWv/#introduction-to-the-box-model>